admin-addon-revisions.php

```diff
@@ -60,6 +60,7 @@ public function onPluginsInitialized() {
     $this->enable([
       'onPageProcessed' => ['onPageProcessed', 0],
       'onAdminTwigTemplatePaths' => ['onAdminTwigTemplatePaths', 0],
+      'onTwigTemplatePaths' => ['onTwigTemplatePaths', 0],
```

Comment:
Removing this line on GRAV Grav v1.6.6 + Admin v1.9.2, still raise the RuntimeException (400)